.github/workflows/main.yml

```diff
@@ -30,14 +24,16 @@ jobs:
 
             - name: 📥 Download deps
               uses: bahmutov/npm-install@v1
-              with:
-                  useLockFile: false
 
-            - name: ▶️ Run validate script
-              run: npm run validate
```

Comment:
I think `npm run validate` also does the prettier formatting check and the TypeScript type checks. I'm open to changing this but really want to keep CI doing the type checking and formatting check